content/general/concepts/front-end/front-end.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'Front-end'
+Description: 'The front-end of a website includes elements that the end user interacts with directly.'
+Subjects:
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Android'
+  - 'Browsers'
+  - 'Debugging'
+  - 'Development'
+  - 'Frameworks'
+  - 'iOS'
```

Comment:
i would actually take iOS, Android, Search, Semantic, Technical Interviews out. let's keep just the tags that are "core":

- 'Browsers'
- 'Frameworks'
- 'React'
- 'jQuery'
-